gensim/sklearn_api/lsimodel.py

```diff
@@ -5,9 +5,19 @@
 # Copyright (C) 2017 Radim Rehurek <radimrehurek@seznam.cz>
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
-"""
-Scikit learn interface for gensim for easy use of gensim with scikit-learn
-Follows scikit-learn API conventions
+"""Scikit learn interface for `gensim.models.lsimodel`.
+
+Follows scikit-learn API conventions to facilitate using gensim along with scikit-learn.
+
+Examples
+--------
+Integrate with sklearn Pipelines:
+
+    >>> model = LsiTransformer(num_topics=15, id2word=id2word)
```

Comment:
Great (examples is the really nice idea), but please be sure, that example executable (i.e. you write all needed imports & define data).

You can check that all works easily with `python -m doctest path/to/file/with/examples.py`